### ISSUE TITLE
identify

### ISSUE THREAD

**Issue by crawfordsm** [crawfordsm/specreduce#2]
output the mapping between pixel position and wavelength

**Comment by jradavenport** [comment]
Will you include any automatic identify functionality?

**Comment by crawfordsm** [comment]
Yes a good start usually to any data reduction algorithm is to see what the IRAF folks did.   

If you have something coded up, that would be great to include.   I'm going to try to go through and clean up the code to make it match with astropy standards although that might take a while so feel free to either include it as a separate file or in specutils.

Let me know if you would like to test it out with the interactive GUI and I can add instructions for that.
